doc/api/errors.md

```diff
@@ -1912,6 +1912,11 @@ removed: v10.0.0
 Used when an invalid character is found in an HTTP response status message
 (reason phrase).
 
+<a id="ERR_HTTP_SOCKET_TIMEOUT"></a>
+### ERR_HTTP_SOCKET_TIMEOUT
```

Comment:
@mcollina change to `ERR_HTTP_SOCKET_TIMEOUT` and add update on `request.setTimeout()`.